src/style.css

```diff
@@ -0,0 +1,41 @@
+@font-face {
+  font-family: Avenir, Times, serif;
+  src: url(./fonts/Avenir.ttc),
+    url(./fonts/Avenir-Book.ttf),
+    url(./fonts/Avenir-Heavy.ttf);
+}
+
+.searchForm {
+  margin: 0 0 20px;
```

Comment:
let's replace these bottom margins with top margins, it help you to handle top offset required for tests, just like this
![Screenshot 2022-11-21 at 16 33 10](https://user-images.githubusercontent.com/75496546/203080950-05cf896b-04f9-4ddf-aae4-5feab6cd2876.png)
```suggestion
  margin-top: 20px;
```